lib/src/main/java/com/auth0/android/jwt/BaseClaim.java

```diff
@@ -57,4 +57,10 @@ public Date asDate() {
     public <T> List<T> asList(Class<T> tClazz) throws DecodeException {
         return Collections.emptyList();
     }
+
+    @Nullable
+    @Override
+    public <T> T asObject(Class<T> tClazz) throws DecodeException {
+        return null;
```

Comment:
Coverage check is complaining this change has not been tested. Please add the missing test [here](https://github.com/auth0/JWTDecode.Android/blob/master/lib/src/test/java/com/auth0/android/jwt/BaseClaimTest.java)

Reply:
Added shouldGetAsObject() test in BaseClaimTest class